.github/workflows/project_thread_update_reminders.yml

```diff
@@ -1,9 +1,10 @@
-# This workflow is currently a stub to allow for testing the
-# project thread update action manually in the GitHub UI.
-# Actions must be present in the main branch to appear there.
 name: Project Thread Update Reminders
 
 on:
+  schedule:
+    # Run every day at 00:00 UTC
+    - cron: "0 0 * * *"
+  # To allow for manual testing
   workflow_dispatch:
```

Comment:
Will this be removed before merge?

Reply:
I'd prefer to keep it, barring any concerns, for the ease of development if we need to make adjustments in the future.

Reply:
FWIW, I've found that another easy way to iterate on actions like this in PRs is to add a push event. For dispatchable PRs, like the Release app workflow, I found success creating a separate workflow that triggers on PR push and in turn dispatches the workflow that needs testing.

That is to say: is there a context where this workflow needs to be dispatched manually for testing outside of when changes are being made in a PR? If so, then we can keep it. If not, then I think removing it helps to clarify the intended usage of the workflow in a self-documenting way.

Just a strong suggestion but not a requirement. We don't have a consistent policy for this or anything, though I think it would be good if we did, specifically to make the workflows more self-documenting as to their use cases.

Reply:
> For dispatchable PRs, like the Release app workflow, I found success creating a separate workflow that triggers on PR push and in turn dispatches the workflow that needs testing.

Oh, of course, this is much better. Thanks for the suggestion!